master/pyext/src/validation/__init__.py

```diff
@@ -241,12 +241,17 @@ def get_sampling(self) -> dict:
                     str(step.multi_scale))
                 sampling_comp['Protocol ID'].append(
                     self.system.orphan_protocols.index(prot)+1)
-                cit = self.system.citations[0].pmid
-                link = 'https://pubmed.ncbi.nlm.nih.gov/'+str(cit)+'/'
-                if step.name:
-                    method_link = '<a href='+link+'>'+str(step.method)+'</a>'
+                cit = str(self.system.citations[0].pmid)
```

Comment:
Note that there's a difference in mmCIF between `.` and `"."`. The former means the data item is not applicable and the latter is the literal string ".". In python-ihm `.` maps to Python `None` and `?` to Python `ihm.unknown`.

Reply:
thanks for the comment! I noticed that in the case of entry 93  the citation was mapped to  `<class 'ihm.__UnknownValue'>`, but since neither '.' or '?' doesn't make sense for the PubMed link I decided to use the explicit string check.